src/PropertyInfo/Extractor/MappingReflectionExtractor.php

```diff
@@ -4,10 +4,10 @@
 
 namespace OpenEuropa\EPoetry\PropertyInfo\Extractor;
 
-use OpenEuropa\EPoetry\Type\ContactPersonIn;
```

Comment:
These files chanced mode `100644 -> 100755` we should restore that everywhere.

Reply:
Done